pandas/tests/series/test_apply.py

```diff
@@ -576,3 +576,13 @@ def f(x):
         result = s.map(f)
         exp = pd.Series(['Asia/Tokyo'] * 25, name='XX')
         tm.assert_series_equal(result, exp)
+
+    @pytest.mark.parametrize("vals,mapping,exp", [
+        (list('abc'), {np.nan: 'not NaN'}, ['not NaN']),
+        (list('abc'), {'string': 'another string'}, ['another string']),
```

Comment:
Your mapping should contain one of the values in the series, so use 'a' instead of 'string'

Reply:
Riiight

Reply:
The tests *did* pass

Just not like they should have